zopkio/test_runner.py

```diff
@@ -377,6 +427,37 @@ def _execute_verification(self):
           test.result = constants.FAILED
           test.exception = e
 
+  def _execute_singletest_verification(self,test):
+
+    if (test.result != constants.SKIPPED
```

Comment:
should this check if the result is passed since if the test failed there's not much point in running the validation